### PR TITLE
Add Python Solution to Daisy Chains (Bronze)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
also added a short editorial

### CODE REVIEW

File: solutions/usaco-1060.mdx [hunk 1]
```diff
@@ -123,5 +126,31 @@ class Main {
...
 				 
 </JavaSection>
+<PySection>
+
+```py
+N = int(input())
+flowers = list(map(int, input().split()))
+ans = 0
+
+def search(i, j):
```
Comment (SansPapyrus683) [inline solutions/usaco-1060.mdx:136]: why not just put this in the loop i mean

File: solutions/usaco-1060.mdx [hunk 2]
```diff
@@ -123,5 +126,31 @@ class Main {
...
+	# find average petals in the range i - j
+	avg_petals = sum(flowers[i:j + 1]) / len(flowers[i : j + 1])
+	# since a flower can't have a decimal amount of petals, we can return if the average is a float 
+	if int(avg_petals) != avg_petals:
+		return False
```
Comment (SansPapyrus683) [inline solutions/usaco-1060.mdx:141]: is this optimization necessary? feel like it'd still pass w/o

File: solutions/usaco-1060.mdx [hunk 3]
```diff
@@ -123,5 +126,31 @@ class Main {
...
+<PySection>
+
+```py
+N = int(input())
+flowers = list(map(int, input().split()))
+ans = 0
+
+def search(i, j):
+	# find average petals in the range i - j
+	avg_petals = sum(flowers[i:j + 1]) / len(flowers[i : j + 1])
```
Comment (SansPapyrus683) [inline solutions/usaco-1060.mdx:138]: at least be consistent w/ the colon spacing, this i don't have a preference

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
oh god i just ate a burrito my mouth is burning

File: solutions/usaco-1060.mdx (hunk 3 above)
Comment (ryanchou-dev) [reply solutions/usaco-1060.mdx:138]: oops

**Review by SansPapyrus683 — APPROVED** [review]
jotaro-kujo-yes

**Review by TruaShamu — APPROVED** [review]

**Review by ar492 — APPROVED** [review]
👍

**Review by maggie-j-liu — APPROVED** [review]

**Review by maggie-j-liu — APPROVED** [review]

**Review by HarryWangATX — APPROVED** [review]

**Pull request merged**